app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -171,13 +219,20 @@ class RefereeProfile extends Component {
             {this.hasPassedTest('field') ? '✓' : '✗'}
           </dd>
         </dl>
-        <h2>
+        <Header as="h2">
           Bio
-        </h2>
+        </Header>
         <p>
-          {bio}
+          {referee.bio}
         </p>
-      </div>
+        {
+          referee.isEditable && (
```

Comment:
Let's put this button at the top of the profile, so refs who are viewing their own profile know they can edit it sooner.